docs/source/how-to-upstream.mdx

```diff
@@ -255,18 +76,18 @@ When `blocking=False`, commands are tracked, and your script will only exit when
 >>> last_command.failed
 ```
 
-### push_to_hub
+## push_to_hub
```

Comment:
For me it's a bit confusing to know what would go here and in how-to-manage. This seems in line with the push methods shown in how-to-upstream

Reply:
Yeah, I definitely see where the confusion is coming from. I'll move the `add`, `commit` and `push` methods to how-to-upstream. The how-to-manage section will be focused on how to create/delete a repository, create branches and clones, and just the `pull` function for updating.